src/SugarModules/modules/supp_SugarRepairs/Classes/Repairs/supp_ReportRepairs.php

```diff
@@ -16,6 +16,29 @@ function __construct()
     }
 
     /**
+     * Prefixes a given report with Broken: to let users know problematic reports
+     * @param $id
+     */
+    public function markReportBroken($id)
```

Comment:
please move these function back to the main repair class. although they are obviously related to certain functionality reports, other repairs may need to use this in their logic.